glassfish-tck-runner/pom.xml

```diff
@@ -215,17 +228,32 @@
         <dependency>
 	        <groupId>org.glassfish.jersey.inject</groupId>
 	        <artifactId>jersey-hk2</artifactId>
-	        <version>3.0.0</version>
+	        <version>3.0.3</version>
         </dependency>
         <dependency>
 	        <groupId>org.glassfish.hk2</groupId>
 	        <artifactId>hk2-locator</artifactId>
-	        <version>3.0.0</version>
+	        <version>3.0.3</version>
```

Comment:
It would be perfect to use consistent formatting (not your mistake, previous version mixed tabs and spaces).

Reply:
done